src/components/grid-aware/ProgramBlock/ProgramBlock.module.css

```diff
@@ -0,0 +1,139 @@
+/* Background bleed */
+
+.bleedWrapper {
+  display: grid;
+  grid-template-columns: var(--bleed-grid-template-columns);
+  margin: auto;
+  margin-bottom: 100px;
+  max-width: var(--bleed-max-width);
+}
+
+/* Override grid item default of auto so that item contents cannot stretch the grid itself */
+.bleedWrapper > * {
+  min-width: 0;
+}
+
+@media (--tablet-and-down) {
+  .bleedWrapper {
+    grid-template-columns: 40px 1fr 40px;
+    margin-bottom: 80px;
+  }
+}
+
+.bleedMainContent {
+  grid-column: 2;
+}
+
+
+/* Main grid */
+
+.gridParent {
+  column-gap: var(--grid-column-gap);
+  display: grid;
+  grid-template-columns: repeat(var(--grid-column-count), minmax(0, 1fr));
+}
+
+.gridParent > * {
+  min-width: 0;
+}
+
+@media (--tablet-and-down) {
+  .gridParent {
+    display: flex;
+    flex-direction: column-reverse;
+  }
+}
+
+
+/* Main grid areas */
+
+.gridAreaText {
+  grid-column: 6/ span 6;
+  grid-row: 1;
+}
+
+@media (--tablet-and-down) {
+  .gridAreaText {
+    padding-top: 16px;
+  }
+}
+
+.gridAreaImage {
+  grid-column: 2 / span 4;
+  grid-row: 1;
+}
+
+/* Left grid area */
+
+.title {
+  font: var(--font-body-large);
+  margin-bottom: 5px;
+  margin-top: 0;
+}
+
+@media (--tablet-and-down) {
+  .title {
+    margin-top: 0;
+  }
+}
+
+.subtitle {
+  font: var(--font-subtitle);
+}
+
+.description {
+  font: var(--font-body-medium);
+  margin-top: 20px;
+}
+
+.description :first-child {
+  margin: 0;
+}
+
+.description > :not(:first-child) {
+  margin-top: 20px;
+}
+
+.description ul {
+  margin-left: 1.5em;
+}
+
+.description ul > li {
+  list-style-position: outside;
+  list-style-type: disc;
+}
+
+.ctaButtonRow {
+  display: flex;
+  flex-flow: row wrap;
+  margin-top: 30px;
+}
+
+.ctaButtonRowItem:not(:last-child) {
+  margin-right: 30px;
```

Comment:
I _really_ want to say that we should just use the new `column-gap` property for flexbox here, since that'll allow us to avoid having to do this annoying `:not(:last-child)` selector. Originally, the [`gap`](https://developer.mozilla.org/en-US/docs/Web/CSS/gap) properties were only for CSS Grid, but it was recently updated so that they could apply to flexbox as well. Firefox started supporting gap for flexbox in 2018, and Chrome and Edge added it in 2020, but Safari only added support a few months ago. Overall, it looks like only 82% of users are on browser versions that support it: https://caniuse.com/flexbox-gap

How about we use `column-gap`, and we get bug reports from people on older browsers, then we go back to this `:not(:last-child)` approach?